From ffb42bb0d87586adac14144b98333eb52808ce99 Mon Sep 17 00:00:00 2001
From: Mehmet Emre Atasever <memre@pardus.org.tr>
Date: Sun, 26 Jun 2011 18:39:04 +0300
Subject: [PATCH] nanorc default settings changes

Some default settings are not so convenient for Pisi Linux nano users.

also fixes bugs (#pb18483, #pb18484)
---
 doc/nanorc.sample.in |   64 +++++++++++++++++++++++++-------------------------
 1 files changed, 32 insertions(+), 32 deletions(-)

diff --git a/doc/nanorc.sample.in b/doc/nanorc.sample.in
index f164c26..24d3c7b 100644
--- a/doc/nanorc.sample.in
+++ b/doc/nanorc.sample.in
@@ -25,17 +25,17 @@
 # set backupdir ""
 
 ## Do backwards searches by default.
-# set backwards
+set backwards
 
 ## Use bold text instead of reverse video text.
-# set boldtext
+set boldtext
 
 ## The characters treated as closing brackets when justifying
 ## paragraphs.  They cannot contain blank characters.  Only closing
 ## punctuation, optionally followed by closing brackets, can end
 ## sentences.
 ##
-# set brackets ""')>]}"
+set brackets ""')>]}"
 
 ## Do case sensitive searches by default.
 # set casesensitive
@@ -60,7 +60,7 @@
 ## searches.  They cannot contain blank characters.  The former set must
 ## come before the latter set, and both must be in the same order.
 ##
-# set matchbrackets "(<[{)>]}"
+set matchbrackets "(<[{)>]}"
 
 ## Use the blank line below the titlebar as extra editing space.
 # set morespace
@@ -139,7 +139,7 @@
 # set smarthome
 
 ## Use smooth scrolling as the default.
-# set smooth
+set smooth
 
 ## Enable soft line wrapping (AKA full line display).
 # set softwrap
@@ -153,7 +153,7 @@
 # set suspend
 
 ## Use this tab size instead of the default; it must be greater than 0.
-# set tabsize 8
+set tabsize 4
 
 ## Convert typed tabs to spaces.
 # set tabstospaces
@@ -234,79 +234,79 @@
 
 
 ## Nanorc files
-# include "@PKGDATADIR@/nanorc.nanorc"
+include "@PKGDATADIR@/nanorc.nanorc"
 
 ## C/C++
-# include "@PKGDATADIR@/c.nanorc"
+include "@PKGDATADIR@/c.nanorc"
 
 ## Makefiles
-# include "@PKGDATADIR@/makefile.nanorc"
+include "@PKGDATADIR@/makefile.nanorc"
 
 ## Cascading Style Sheets
-# include "@PKGDATADIR@/css.nanorc"
+include "@PKGDATADIR@/css.nanorc"
 
 ## Debian files
-# include "@PKGDATADIR@/debian.nanorc"
+include "@PKGDATADIR@/debian.nanorc"
 
 ## Gentoo files
-# include "@PKGDATADIR@/gentoo.nanorc"
+include "@PKGDATADIR@/gentoo.nanorc"
 
 ## HTML
-# include "@PKGDATADIR@/html.nanorc"
+include "@PKGDATADIR@/html.nanorc"
 
 ## PHP
-# include "@PKGDATADIR@/php.nanorc"
+include "@PKGDATADIR@/php.nanorc"
 
 ## TCL
-# include "@PKGDATADIR@/tcl.nanorc"
+include "@PKGDATADIR@/tcl.nanorc"
 
 ## TeX
-# include "@PKGDATADIR@/tex.nanorc"
+include "@PKGDATADIR@/tex.nanorc"
 
 ## Quoted emails (under e.g. mutt)
-# include "@PKGDATADIR@/mutt.nanorc"
+include "@PKGDATADIR@/mutt.nanorc"
 
 ## Patch files
-# include "@PKGDATADIR@/patch.nanorc"
+include "@PKGDATADIR@/patch.nanorc"
 
 ## Manpages
-# include "@PKGDATADIR@/man.nanorc"
+include "@PKGDATADIR@/man.nanorc"
 
 ## Groff
-# include "@PKGDATADIR@/groff.nanorc"
+include "@PKGDATADIR@/groff.nanorc"
 
 ## Perl
-# include "@PKGDATADIR@/perl.nanorc"
+include "@PKGDATADIR@/perl.nanorc"
 
 ## Python
-# include "@PKGDATADIR@/python.nanorc"
+include "@PKGDATADIR@/python.nanorc"
 
 ## Ruby
-# include "@PKGDATADIR@/ruby.nanorc"
+include "@PKGDATADIR@/ruby.nanorc"
 
 ## Java
-# include "@PKGDATADIR@/java.nanorc"
+include "@PKGDATADIR@/java.nanorc"
 
 ## Fortran
-# include "@PKGDATADIR@/fortran.nanorc"
+include "@PKGDATADIR@/fortran.nanorc"
 
 ## Objective-C
-# include "@PKGDATADIR@/objc.nanorc"
+include "@PKGDATADIR@/objc.nanorc"
 
 ## OCaml
-# include "@PKGDATADIR@/ocaml.nanorc"
+include "@PKGDATADIR@/ocaml.nanorc"
 
 ## AWK
-# include "@PKGDATADIR@/awk.nanorc"
+include "@PKGDATADIR@/awk.nanorc"
 
 ## Assembler
-# include "@PKGDATADIR@/asm.nanorc"
+include "@PKGDATADIR@/asm.nanorc"
 
 ## Bourne shell scripts
-# include "@PKGDATADIR@/sh.nanorc"
+include "@PKGDATADIR@/sh.nanorc"
 
 ## POV-Ray
-# include "@PKGDATADIR@/pov.nanorc"
+include "@PKGDATADIR@/pov.nanorc"
 
 ## XML-type files
-# include "@PKGDATADIR@/xml.nanorc"
+include "@PKGDATADIR@/xml.nanorc"
-- 
1.7.5.4

